docs/voice-transformers/alien-voice/the-on-off-button.md

```diff
@@ -6,9 +6,10 @@ The Nucleo board has a user programmable push button. We will now use it as an O
 
 The idea is to use the push button to call an asynchronous routine in our code. To do that, we need to configure the button to trigger an interrupt and then we need to catch the interrupt in our code.
 
-Go into CubeMX by clicking on the `ioc` file in your alien voice project; in the left panel click on "System &gt; NVIC" and enable the line "EXTI line 4 to 15" by checking the corresponding checkmark. Indeed PC13 is linked to EXTI13 in the hardware of the microcontroler because interrupts are very fast access to the core of the system.
+Go into CubeMX by clicking on the `ioc` file in your alien voice project; in the left panel click on "System &gt; NVIC" and enable the line "EXTI line 4 to 15" by checking the corresponding checkmark. Indeed PC13 is linked to EXTI13 in the hardware of the microcontroler because interrupts provide a very fast access to the core of the system.
+<!-- the logic of above sentence is not clear to me -->
```

Comment:
what do you mean by "because interrupts provide a very fast access to the core of the system"? Does this explain why PC13 is linked to EXTI13? Or does it explain why we want to use interrupts in the first place?

Reply:
corrected thanks